specs/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/streams/kafka/publish.client.topic.space/client.rpt

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+connect "zilla://streams/kafka0"
+         option zilla:window 8192
+         option zilla:transmission "duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                             .typeId(zilla:id("kafka"))
+                             .merged()
+                                 .capabilities("PRODUCE_ONLY")
+                                 .topic("mqtt-clients")
+                                 .partition(-1, -2)
+                                 .ackMode("LEADER_ONLY")
+                                 .build()
+                             .build()}
+
+connected
+
+write notify PRODUCE_CONNECTED
+write zilla:data.ext ${kafka:dataEx()
+                            .typeId(zilla:id("kafka"))
+                            .merged()
+                                .deferred(0)
+                                .partition(-1, -1)
+                                .key("/clients/client-2/sensors/one")
+                                .hashKey("client-2")
+                                .header("zilla:filter", "clients")
+                                .header("zilla:filter", "client-2")
+                                .header("zilla:filter", "sensors")
+                                .header("zilla:filter", "one")
```

Comment:
Is this missing a leading `.header("zilla:filter", "")` to represent the empty segment before the leading `/` in the mqtt topic name?

Perhaps it would be more readable to include the leading `/` for each segment in the `zilla:filter` header value, then omit the leading `/` when not present at the start of the mqtt topic name to differentiate instead of needing a leading empty `""` header value?  That would be a backwards incompatible change, but if we want to do it we should do it sooner than later. Let's discuss.